pkgs/development/python-modules/pysol_cards/default.nix

```
@@ -0,0 +1,25 @@
+{ stdenv, fetchPypi, python }:
```

since this is in python-modules, you should import the individual packages from the python package set, or else everything will be fixed to python2, and it create incoherence in some environments.
```suggestion
{ stdenv, fetchPypi, buildPythonPackage, pbr, random2, six }:
```